apps/products-feed/src/pages/api/feed/[url]/[channel]/[cursor]/generate-chunk.ts

```diff
@@ -0,0 +1,99 @@
+import { AuthData } from "@saleor/app-sdk/APL";
+import { NextApiHandler } from "next";
+
+import { createInstrumentedGraphqlClient } from "../../../../../../lib/create-instrumented-graphql-client";
+import { createLogger } from "../../../../../../logger";
+import { createS3ClientFromConfiguration } from "../../../../../../modules/file-storage/s3/create-s3-client-from-configuration";
+import { uploadFile } from "../../../../../../modules/file-storage/s3/upload-file";
+import {
+  getChunkDownloadUrl,
+  getChunkFileName,
+} from "../../../../../../modules/file-storage/s3/urls-and-names";
+import { FeedXmlBuilder } from "../../../../../../modules/google-feed/feed-xml-builder";
+import { fetchVariants } from "../../../../../../modules/google-feed/fetch-product-data";
+import { GoogleFeedSettingsFetcher } from "../../../../../../modules/google-feed/get-google-feed-settings";
+import { productVariantToProxy } from "../../../../../../modules/google-feed/product-variant-to-proxy";
+
```

Comment:
suggestion: let's use alias `@` for imports, I think auto imports from lsp might have added these incorrectly